data-sources/gigamon-appinsights/config.yml

```diff
@@ -0,0 +1,28 @@
+id: gigamon
```

Comment:
We probably shouldn't have both data-sources with the same name/id, since they will appear side-by-side in NR1.